tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -268,5 +268,257 @@ describe('Test Fleet access with RBAC with custom roles using Standard User', {
+      ]
+    });
+
+    cy.createRoleTemplate({
+      roleType: roleTypeTemplate,
+      roleName: customRoleName_2,
+      rules: [
+        { resource: "fleetworkspaces", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        { resource: "gitrepos", verbs: ["list", "create"]},
+        { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+      ]
+    });
+
+    cy.createRoleTemplate({
+      roleType: roleTypeTemplate,
+      roleName: customRoleName_3,
+      rules: [
+        { resource: "fleetworkspaces", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        { resource: "gitrepos", verbs: ["list", "create", "update", "get"]},
+        { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+      ]
+    });
+      
+    cy.createRoleTemplate({
+      roleType: roleTypeTemplate,
+      roleName: customRoleName_4,
+      rules: [
+        { resource: "fleetworkspaces", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        { resource: "gitrepos", verbs: ["list", "delete"]},
+        { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+      ]
+    });
+  })
+
+  // Pls note this is anti-pattern: 
+  // https://docs.cypress.io/guides/references/best-practices#Using-after-Or-afterEach-Hooks
+  // Done here for demonstration. Better to set before if needed.
+  after('Deleting Fleet repos, Roles, Users', () => {
+    cy.login();
+    cy.deleteAllFleetRepos();
+    // Delete Standard Users
+    const stdusers = ["std-user-46", "std-user-47", "std-user-48", "std-user-50"];
+    stdusers.forEach(user => {
+      cy.deleteUser(user);
+    })
+    // Delete Custom Roles
+    const customRoles = [customRoleName_1, customRoleName_2, customRoleName_3, customRoleName_4];
+    customRoles.forEach(role => {
+      cy.deleteRole(role, roleTypeTemplate.toUpperCase());
+    })
+  })
+
+
+  qase(46,
+    it('Fleet-46: Test "Standard-user" | Custom Role | Fleetworkspaces, Bundles = [ALL] | GitRepos = [List]', { tags: '@fleet-46' }, () => {
+      
+      const stduser = "std-user-46"
+      
+      // Create "Standard User"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(stduser, uiPassword);
+
+      // Assign role to the created user
+      cy.assignRoleToUser(stduser, customRoleName_1)
+      
+      // Logout as admin and login as other user
+      cypressLib.logout();
+      cy.login(stduser, uiPassword);
+
+      // CAN go to Continuous Delivery Dashboard and "list" gitrepos
+      cy.accesMenuSelection('Continuous Delivery', 'Dashboard');
+      cy.get("div[data-testid='collapsible-card-fleet-local']").contains(repoName).should('be.visible');
+      cy.get("div[data-testid='collapsible-card-fleet-default']").contains(repoNameDefault).should('be.visible');
+      
+      // CHECKS IN  FLEET-DEFAULT
```

Comment:
```suggestion
      // CHECKS IN FLEET-DEFAULT
```

Reply:
Thanks, addressed in `3f474bf958123b38439a2f4d1683ac3dcf0b3f16`